packages/suite/src/actions/settings/deviceSettingsActions.ts

```diff
@@ -77,7 +77,11 @@ export const changeWipeCode =
         });
         if (result.success) {
             if (!skipSuccessToast) {
-                dispatch(notificationsActions.addToast({ type: 'wipe-code-changed' }));
+                if (params.remove) {
+                    dispatch(notificationsActions.addToast({ type: 'wipe-code-removed' }));
+                } else {
+                    dispatch(notificationsActions.addToast({ type: 'wipe-code-changed' }));
+                }
```

Comment:
Simplification:
```suggestion
dispatch(
    notificationsActions.addToast({ type: remove ? 'wipe-code-removed' : 'wipe-code-changed' })
);
```

(`params` can be destructured on line 66: `{remove}`)